docs/core/testing/unit-testing-mstest-configure.md

```diff
@@ -100,3 +100,101 @@ Each element of the file is optional because it has a default value.
+
+#### Output settings
+
+| Entry | Default | Description |
+|-------|---------|-------------|
+| captureTrace | false | Capture text messages coming from the `Console.Write*`, `Trace.Write*`, and `Debug.Write*` APIs that will be associated to the current running test. |
+
+#### Parallelism settings
+
+| Entry | Default | Description |
+|-------|---------|-------------|
+| enabled | false | Enable test parallelization. |
+| scope | class | The scope of parallelization. You can set it to `method`. The default, `class`, corresponds to running all tests of a given class sequentially but multiple classes in parallel. |
+| workers | 0 | The number of threads/workers to be used for parallelization. The default value maps to the number of processors on the current machine. |
+
+#### Execution settings
+
+| Entry | Default | Description |
+|-------|---------|-------------|
+| considerEmptyDataSourceAsInconclusive | false | When set to `true`, an empty data source is considered as inconclusive. |
+| considerFixturesAsSpecialTests | false | To display `AssemblyInitialize`, `AssemblyCleanup`, `ClassInitialize`, `ClassCleanup` as individual entries in Visual Studio and Visual Studio Code `Test Explorer` and _.trx_ log, set this value to **true**. |
+| mapInconclusiveToFailed | false | If a test completes with an inconclusive status, it's mapped to the skipped status in **Test Explorer**. If you want inconclusive tests to be shown as failed, set the value to **true**. |
+| mapNotRunnableToFailed | true | A value indicating whether a not runnable result is mapped to failed test. |
+| treatClassAndAssemblyCleanupWarningsAsErrors | false | To see your failures in class cleanups as errors, set this value to **true**. |
+| treatDiscoveryWarningsAsErrors | false | To report test discovery warnings as errors, set this value to **true**. |
+
+#### Timeout settings
+
+| Entry | Default | Description |
+|-------|---------|-------------|
+| assemblyCleanup | 0 | Specify globally the timeout to apply on each instance of assembly cleanup method. |
+| assemblyInitialize | 0 | Specify globally the timeout to apply on each instance of assembly initialize method. |
+| classCleanup | 0 | Specify globally the timeout to apply on each instance of class cleanup method. |
+| classInitialize | 0 | Specify globally the timeout to apply on each instance of class initialize method. |
+| test | 0 | Specify globally the test timeout. |
+| testCleanup | 0 | Specify globally the timeout to apply on each instance of test cleanup method. |
+| testInitialize | 0 | Specify globally the timeout to apply on each instance of test initialize method. |
+| useCooperativeCancellation | false | When set to `true`, in case of timeout, MSTest will only trigger cancellation of the `CancellationToken` but will not stop observing the method. This is more performant but relies on user to correctly flow the token through all paths. |
```

Comment:
```suggestion
| useCooperativeCancellation | false | When set to `true`, in case of timeout, MSTest will only trigger cancellation of the `CancellationToken` but will not stop observing the method. This behavior is more performant but relies on the user to correctly flow the token through all paths. |
```